Documentation/Links.md

```diff
@@ -0,0 +1,59 @@
+# SerenityOS-related pages across the web
+
+This is a roughly categorized list of pages relating to SerenityOS and its subprojects across the web. Many of these pages are "unofficial" in that some single person created them and/or maintains them independently; but we only list pages here that are widely recognized and of good quality. If you want your Serenity related page to be listed, just create a pull request adding it and we can discuss.
+
+## General
+
+-   **[serenityos.org](https://serenityos.org)**
+    -   [Manpages](https://man.serenityos.org/)
+    -   [1st](https://www.serenityos.org/happy/1st/), [2nd](https://www.serenityos.org/happy/2nd/), [3rd](https://www.serenityos.org/happy/3rd/), and [4th](https://www.serenityos.org/happy/4th/) Birthday Post
+    -   [Emoji](https://emoji.serenityos.org/)
+-   [Ports](https://ports.serenityos.net/)
+-   [Try it out Online!](https://copy.sh/v86/?profile=serenity)
+-   [SerenityOS on English Wikipedia](https://en.wikipedia.org/wiki/SerenityOS). This page also links to the SerenityOS articles in many other languages.
+
+## Development
+
+-   [GitHub Organization](https://github.com/SerenityOS) and [GitHub Repositories](https://github.com/orgs/SerenityOS/repositories)
+-   [Changelog](https://changelog.serenityos.org/)
+-   [Issues Found by OSS-Fuzz Continous Fuzzing](https://bugs.chromium.org/p/oss-fuzz/issues/list?q=label:Proj-serenity)
+-   [Azure CI Overview](https://dev.azure.com/SerenityOS/SerenityOS/_build)
+-   [SonarCloud Static Analysis](https://sonarcloud.io/project/overview?id=SerenityOS_serenity)
+-   [libjs.dev](https://libjs.dev/)
+    -   [Try LibJS!](https://libjs.dev/repl/)
```

Comment:
Maybe this should make clear that you are about to try LibJS online (like with copy.sh above)